x-pack/plugins/security_solution/public/timelines/components/open_timeline/timelines_table/actions_columns.tsx

```diff
@@ -149,10 +149,9 @@ export const getActionsColumns = ({
     'data-test-subj': 'create-rule-from-timeline',
     available: () => actionTimelineToShow.includes('createRule') && onCreateRule != null,
   };
-
   return [
     {
-      width: '80px',
+      width: '150px',
```

Comment:
super minor comment, but I wonder if you could use the same `const { kibanaSecuritySolutionsPrivileges } = useUserPrivileges();` code here and conditionally show 80px or 150px depending on the `crud` privilege?

Reply:
Thanks Philippe for reviewing! 
Updated. `useUserPrivileges()` is a ReactHook and `getActionsColumns` is not a React function, hence cannot directly add the check there. Now privilege is checked at a higher level and passed down as props.

Reply:
yup looks good! :)